pages/main-page.ts

```diff
@@ -0,0 +1,22 @@
+export class MainPage {
+  readonly page: Page;
+  constructor(page: Page) {
+    this.page = page;
+    this.setStorage = page.getByRole('link', { name: 'Storage' });
+    this.setUsers = page.getByRole('link', { name: 'Users' });
+    this.installButton = page.getByRole("button", { name: "Install", exact: true });
+    this.continueButton = page.getByRole("button", { name: "Continue" });
```

Comment:
this actually belongs to a different page, so it is more clear to create different POM for it.

Reply:
Ok, we have a ticket to add this POM, so recover it.